src/types/getters.ts

```diff
@@ -72,114 +72,73 @@ import { SheetViewPlugin } from "../plugins/ui_stateful/sheetview";
-  ConditionalFormatGetters &
-  FigureGetters &
+export type CoreGetters = PluginGetters<typeof SheetPlugin> &
+  PluginGetters<typeof HeaderSizePlugin> &
+  PluginGetters<typeof HeaderVisibilityPlugin> &
+  PluginGetters<typeof CellPlugin> &
+  PluginGetters<typeof MergePlugin> &
+  PluginGetters<typeof BordersPlugin> &
+  PluginGetters<typeof ChartPlugin> &
+  PluginGetters<typeof ImagePlugin> &
+  PluginGetters<typeof FigurePlugin> &
   RangeAdapterGetters &
-  FiltersGetters &
-  SettingsGetters &
-  HeaderGroupingGetters &
-  DataValidationGetters;
-
-type AutofillGetters = Pick<AutofillPlugin, GetterNames<typeof AutofillPlugin>>;
-type AutomaticSumGetters = Pick<AutomaticSumPlugin, GetterNames<typeof AutomaticSumPlugin>>;
-type ClipboardGetters = Pick<ClipboardPlugin, GetterNames<typeof ClipboardPlugin>>;
-type EditionGetters = Pick<EditionPlugin, GetterNames<typeof EditionPlugin>>;
-type EvaluationGetters = Pick<EvaluationPlugin, GetterNames<typeof EvaluationPlugin>>;
-type EvaluationChartGetters = Pick<
-  EvaluationChartPlugin,
-  GetterNames<typeof EvaluationChartPlugin>
->;
-type EvaluationConditionalFormatGetters = Pick<
-  EvaluationConditionalFormatPlugin,
-  GetterNames<typeof EvaluationConditionalFormatPlugin>
->;
-type FindAndReplaceGetters = Pick<FindAndReplacePlugin, GetterNames<typeof FindAndReplacePlugin>>;
-type HeaderVisibilityIUIGetters = Pick<
-  HeaderVisibilityUIPlugin,
-  GetterNames<typeof HeaderVisibilityUIPlugin>
->;
-type HighlightGetters = Pick<HighlightPlugin, GetterNames<typeof HighlightPlugin>>;
-type CustomColorsGetters = Pick<CustomColorsPlugin, GetterNames<typeof CustomColorsPlugin>>;
-type RendererGetters = Pick<RendererPlugin, GetterNames<typeof RendererPlugin>>;
-type SelectionGetters = Pick<GridSelectionPlugin, GetterNames<typeof GridSelectionPlugin>>;
-type SelectionInputGetters = Pick<
-  SelectionInputsManagerPlugin,
-  GetterNames<typeof SelectionInputsManagerPlugin>
->;
-type CollaborativeGetters = Pick<CollaborativePlugin, GetterNames<typeof CollaborativePlugin>>;
-type SortGetters = Pick<SortPlugin, GetterNames<typeof SortPlugin>>;
-type UIOptionsGetters = Pick<UIOptionsPlugin, GetterNames<typeof UIOptionsPlugin>>;
-type SheetUIGetters = Pick<SheetUIPlugin, GetterNames<typeof SheetUIPlugin>>;
-type ViewportGetters = Pick<SheetViewPlugin, GetterNames<typeof SheetViewPlugin>>;
-type CellPopoverPluginGetters = Pick<CellPopoverPlugin, GetterNames<typeof CellPopoverPlugin>>;
-type FilterEvaluationGetters = Pick<
-  FilterEvaluationPlugin,
-  GetterNames<typeof FilterEvaluationPlugin>
->;
-type SplitToColumnsGetters = Pick<SplitToColumnsPlugin, GetterNames<typeof SplitToColumnsPlugin>>;
-type UIRowSizeGetters = Pick<UIRowSizePlugin, GetterNames<typeof UIRowSizePlugin>>;
-type EvaluationDataValidationPluginGetters = Pick<
-  EvaluationDataValidationPlugin,
-  GetterNames<typeof EvaluationDataValidationPlugin>
->;
+  PluginGetters<typeof ConditionalFormatPlugin> &
+  PluginGetters<typeof HistoryPlugin> &
```

Comment:
this was not is the core getters before, is it on purpose?

Reply:
good catch :+1: